deploy/hs-delete-custom-cmo-config/02-hs-delete-custom-cmo.ClusterRole.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: hs-delete-custom-cmo-config
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
```

Comment:
Should this be `syncset` instead of `configmaps`

Reply:
You're right, got confused, fixed now